test/distributed/pipeline/sync/test_microbatch.py

```diff
@@ -44,7 +44,7 @@ def test_batch_non_atomic():
 
 def test_batch_call():
     a = Batch(torch.tensor(42))
-    b = Batch((torch.tensor(42), torch.tensor(21)))
+    b = Batch((torch.tensor(42), torch.tensor(21)), is_single_sequence=True)
```

Comment:
it seems we do not have a similar test with is_single_sequence=False (that tests .call method) here.